chall03/ssacrist.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import sys
+
+def byebye():
```

Comment:
You need to use more "serious" naming for functions and variables :)

Reply:
Well, I undestand you and maybe my level poor english is not a help to me, but... I code as I am. But I'll keep that in mind.

Reply:
I didn't mean this in a bad way, when I was a student I used to use funny variable and function names too all the time. But the day you'll be in a company, people won't like that, it's better to have descriptive and serious variable and function names and comments.